code/src/App.js

```diff
@@ -1,9 +1,145 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { QuestionSection } from './QuestionSection'
 
 export const App = () => {
+
+  const groups = ['0-100', '100-200', '500+']
+  const [grams, setGrams] = useState('')
+  const [sugaraddict, setSugaraddict] = useState(false)
+  const [favoriteCandy, setFavoriteCandy] = useState('')
+  const [name, setName] = useState('')
+  const [showSummary, setShowSummary] = useState(false)
+
+  const handleSubmit = event => {
+    event.preventDefault()
+    setShowSummary(true)
+  }
+
   return (
-    <div>
-      Find me in src/app.js!
+    <div className="App">
+
+      <header>
+        <h2>
+          WICH TYPE OF
+        </h2>
+        <h1>
+          CANDY
+        </h1>
+        <h2>
+          ARE YOU?
+        </h2>
+      </header>
+
+      {!showSummary && (
+        <form onSubmit={handleSubmit}>
+
+          {/* HERE YOU CAN CHOOSE A GROUP WITH RADIOBUTTONS */}
```

Comment:
The user that will take the survey wont look in here. So the comments is for developers (you or others) and not the end user